AK Zeus/app/src/main/java/ayds/zeus/songinfo/moredetails/fulllogic/DataBase.kt

```diff
@@ -0,0 +1,66 @@
+package ayds.zeus.songinfo.moredetails.fulllogic
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.util.Log
+
+class DataBase(context: Context): SQLiteOpenHelper(context,"dictionary.db", null, 1) {
+
+    fun saveArtist(artist: String, info: String) {
+        val dataBase = this.writableDatabase
+        val contentValues = getArtistContentValues(artist, info)
+        dataBase.insert("artists", null, contentValues)
+    }
+
+    fun getInfo(artist: String): String? {
+        val cursor = getNewArtistCursor(artist)
+        val items = getCursorItems(cursor)
+        cursor.close()
+        return if (items.isEmpty())
+            null
+        else
+            items[0]
```

Comment:
Se puede simplificar
```suggestion
        return items.firstOrNull()
```